code/__DEFINES/robots.dm

```diff
@@ -27,6 +27,7 @@
 #define BOT_NAV				15	// computing navigation
 #define BOT_WAIT_FOR_NAV	16	// waiting for nav computation
 #define BOT_NO_ROUTE		17	// no destination beacon found (or no route)
+#define BOT_SHOWERSTANCE	18	// cleaning unhygienic humans
```

Comment:
These defines are used to report a bot's status to players accessing them remotely (via PDA or AI's Botcall interface).
I recommend either removing this and just using "BOT_CLEANING", which will report a status of "Cleaning" to players, OR if you want to add something witty, go to the lines I am linking below and add a status message at the **end** of that list.
https://github.com/tgstation/tgstation/blob/56bdde9594ab5ec4d452cc47657e23b26dd95060/code/modules/mob/living/simple_animal/bot/bot.dm#L81-L85

Reply:
thanks

Reply:
that should probably be noted at the defines